src/options.h

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright (c) 2017-2020, Intel Corporation
```

Comment:
It needs to be only 2020, as it's a new file.

Reply:
It's not new: [https://github.com/intel/yarpgen/blob/master/src/options.h]

Reply:
ok.